src/traces/sankey/render.js

```diff
@@ -444,11 +444,11 @@ module.exports = function(svg, styledData, layout, callbacks) {
         .append('g')
         .classed('sankeyLinks', true)
         .style('fill', 'none')
-        .style('transform', linksTransform);
```

Comment:
The tweening effect when switching between horizontal and vertical layout was a bit like a circus attraction, and interestingly, the switch to `attr` made it noticeably worse, crossing into creepy territory. I'm reworking a chunk of code so that links and nodes don't "fly" on separate arcs but they remain joint instead. It also stops the node rectangle shape morphing too. As I think this PR stands on its legs though (fixing the error) it'll be a separate PR based on this one.